okta_rules/okta_account_support_access.yml

```diff
@@ -0,0 +1,76 @@
+AnalysisType: rule
+Filename: okta_account_support_access.py
+RuleID: Okta.Support.Access
+DisplayName: Okta Support Access Granted
+Enabled: true
+LogTypes:
+  - Okta.SystemLog
+Tags:
+  - Identity & Access Management
```

Comment:
Should we add the DataModel tag since this rule requires use of the data model? Or should we update the rule to not require the data model?